common/app/common/configuration.scala

```diff
@@ -34,7 +34,17 @@ object Environment extends GuLogging {
 
   // Prefer env vars over install vars over default
   private[this] def get(name: String, default: String): String = {
-    sys.env.get(name.toUpperCase).orElse(installVars.get(name)).getOrElse(default)
+    sys.env.get(name.toUpperCase) match {
+      case Some(env) =>
+        log.info(s"Resolved $name from environment variable")
+        env
+      case _ if installVars.contains(name) =>
+        log.info(s"Resolved $name from install_vars")
+        installVars(name)
+      case _ =>
+        log.info(s"Resolved $name from default value")
```

Comment:
Might a prefix make it easier to search for these messages as a group?

```suggestion
        log.info(s"Environment lookup: resolved $name from environment variable")
        env
      case _ if installVars.contains(name) =>
        log.info(s"Environment lookup: resolved $name from install_vars")
        installVars(name)
      case _ =>
        log.info(s"Environment lookup: resolved $name from default value")
```

Reply:
Good idea! Worth also mentioning we're producing a `logger_name` marker and can filter for these logs with `logger_name.keyword = common.Environment`.